.github/workflows/build.yml

```diff
@@ -66,6 +72,7 @@ jobs:
         cmake --build ${{ steps.strings.outputs.build-output-dir }} --config ${{ matrix.build_type }}
 
     - name: Lint
+      if: matrix.enable_runtime == 'OFF'
```

Comment:
Cool, can you file an issue for this? I want to make sure the tt-mlir runtime sources still get linted.

Reply:
https://github.com/tenstorrent/tt-mlir/issues/189